### PR TITLE
DAOS-16310 test: consolidate il and ioil tags

### PR DESCRIPTION
**Pull request by daltonbohning**
Update il -> ioil for consistency

Skip-unit-tests: true
Skip-fault-injection-test: true

Required-githooks: true

### Before requesting gatekeeper:

* [ ] Two review approvals and any prior change requests have been resolved.
* [ ] Testing is complete and all tests passed or there is a reason documented in the PR why it should be force landed and forced-landing tag is set.
* [ ] `Features:` (or `Test-tag*`) commit pragma was used or there is a reason documented that there are no appropriate tags for this PR.
* [ ] Commit messages follows the guidelines outlined [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [ ] Any tests skipped by the ticket being addressed have been run and passed in the PR.

### Gatekeeper:

* [ ] You are the appropriate gatekeeper to be landing the patch.
* [ ] The PR has 2 reviews by people familiar with the code, including appropriate owners.
* [ ] Githooks were used. If not, request that user install them and check copyright dates.
* [ ] Checkpatch issues are resolved.  Pay particular attention to ones that will show up on future PRs.
* [ ] All builds have passed.  Check non-required builds for any new compiler warnings.
* [ ] Sufficient testing is done. Check feature pragmas and test tags and that tests skipped for the ticket are run and now pass with the changes.
* [ ] If applicable, the PR has addressed any potential version compatibility issues.
* [ ] Check the target branch.   If it is master branch, should the PR go to a feature branch?  If it is a release branch, does it have merge approval in the JIRA ticket.
* [ ] Extra checks if forced landing is requested
  * [ ] Review comments are sufficiently resolved, particularly by prior reviewers that requested changes.
  * [ ] No new NLT or valgrind warnings.  Check the classic view.
  * [ ] Quick-build or Quick-functional is not used.
* [ ] Fix the commit message upon landing. Check the standard [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments). Edit it to create a single commit. If necessary, ask submitter for a new summary.

### CODE REVIEW

File: src/tests/ftest/ior/intercept_messages_pil4dfs.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def test_ior_intercept_messages_pil4dfs(self):
 
         :avocado: tags=all,full_regression
         :avocado: tags=hw,medium
-        :avocado: tags=daosio,dfuse,il,ior,ior_intercept,pil4dfs
```
Comment (daltonbohning) [inline src/tests/ftest/ior/intercept_messages_pil4dfs.py:36]: @wiliamhuang I'm not sure why for some of these that had `il` and `pil4dfs`, but do they need `ioil`? Or JUST `pil4dfs`?

Comment (wiliamhuang) [reply src/tests/ftest/ior/intercept_messages_pil4dfs.py:36]: It needs only "pil4dfs" here.
I probably thought "il" including "ioil" and "pil4dfs" previously.

Comment (daltonbohning) [reply src/tests/ftest/ior/intercept_messages_pil4dfs.py:36]: We could make il include both ioil and pil4dfs if that is useful, but it's not 100% the case today

Comment (wiliamhuang) [reply src/tests/ftest/ior/intercept_messages_pil4dfs.py:36]: Does "dfuse" include both "ioil" and "pil4dfs" already?

Comment (daltonbohning) [reply src/tests/ftest/ior/intercept_messages_pil4dfs.py:36]: I'm not sure... Let me check and update if needed. So in total would you be okay with these?
- dfuse - dfuse-related tests
- ioil - specifically dfuse + ioil tests
- pil4dfs - specifically dfuse + pil4dfs tests

Comment (wiliamhuang) [reply src/tests/ftest/ior/intercept_messages_pil4dfs.py:36]: Yes. Sounds good to me. Thank you very much!

**Review by wiliamhuang — APPROVED** [review]

**Review by shimizukko — APPROVED** [review]

**Comment by daltonbohning** [comment]
Failures in https://build.hpdd.intel.com/blue/organizations/jenkins/daos-stack%2Fdaos/detail/PR-14881/3/tests
mu_perms.py - https://daosio.atlassian.net/browse/DAOS-16301
pil4dfs_fio.py - https://daosio.atlassian.net/browse/DAOS-16315

**Review by phender — APPROVED** [review]

**Pull request merged**